src/database-event-rule.ts

```diff
@@ -0,0 +1,183 @@
+  /** Database failure event: instance failure, etc. */
+  FAILURE = 'failure',
+  /** Database low storage event: storage exhausted, etc. */
+  LOW_STORAGE = 'low storage',
+  /** Database maintenance event: instance patched, version upgrade, etc. */
+  MAINTENANCE = 'maintenance',
+  /** Database maintenance failure event: update of Oracle time zone failed. */
+  MAINTENANCE_FAILURE = 'maintenance, failure',
+  /** Database maintenance notification event: time zone update, etc. */
+  MAINTENANCE_NOTIFICATION = 'maintenance, notification',
+  /**
+   * Database notification event: patching delayed, operator issued
+   * notification, exceeding best practices, etc. */
+  NOTIFICATION = 'notification',
+  /** Database read replica event: replication started, stopped, etc. */
+  READ_REPLICA = 'read replica',
+  /** Database recovery event: recovery started, complete. etc. */
+  RECOVERY = 'recovery',
+  /** Database restoration event: restored instance. */
+  RESTORATION = 'restoration',
+  /** Database security event: decrypting HSM password to update instance. */
+  SECURITY = 'security',
+  /** Database security patching event: system update available. */
+  SECURITY_PATCHING = 'security patching',
+}
+
+/**
+ * Properties for defining a database EventBridge rule.
+ *
+ * @see {@link RuleProps}
+ */
+export interface DatabaseEventRuleProps {
+  /**
+   * The scope to use if the source of the rule and its target are in different Stacks
+   * (but in the same account & region).
+   * This helps dealing with cycles that often arise in these situations.
+   *
+   * @default - none (the main scope will be used, even for cross-stack Events)
+   */
+  readonly crossStackScope?: Construct;
+
+  /**
+   * Database instance to monitor.
+   */
+  readonly database: IDatabaseInstance;
+
+  /**
+   * A description of the rule's purpose.
+   *
+   * @default - No description
+   */
+  readonly description?: string;
+
+  /**
+   * Indicates whether the rule is enabled.
+   *
+   * @default true
+   */
+  readonly enabled?: boolean;
+
+  /**
+   * The event bus to associate with this rule.
+   *
+   * @default - The default event bus.
+   */
+  readonly eventBus?: IEventBus;
+
+  /**
+   * Event categories to include in the event filter.
+   *
+   * @remarks
+   * Set to an empty array include all event categories.
+   *
+   * @default [AVAILABILITY, FAILOVER]
+   */
+  readonly eventCategories?: DatabaseEventCategory[];
+
+  /**
+   * Event ids to include in the event filter.
+   *
+   * @example ['RDS-EVENT-0004', 'RDS-EVENT-0006', etc.]
+   *
+   * @default No additional filtering
+   */
+  readonly eventIds?: string[];
+
+  /**
+   * A name for the rule.
+   *
+   * @default AWS CloudFormation generates a unique physical ID.
+   */
+  readonly ruleName?: string;
+
+  /**
+   * The schedule or rate (frequency) that determines when EventBridge
+   * runs the rule.
+   *
+   * You must specify this property, the `eventPattern` property, or both.
```

Comment:
`eventPattern` doesn't exist in this interface. Though, I understand that that is kind of the whole point of this construct... might want to just revise this documentation